traveler-ios-ui/TravelerKitUI/ViewControllers/QueryItemViewController.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  QueryItemViewController.swift
+//  TravelerKitUI
+//
+//  Created by Ata Namvari on 2019-10-07.
+//  Copyright © 2019 Guestlogix Inc. All rights reserved.
+//
+
+import Foundation
+import TravelerKit
+
+public protocol QueryItemViewControllerDelegate {
+    func queryItemViewController(_ controller: QueryItemViewController, didFinishWith purchaseForm: PurchaseForm)
+}
+
+public class QueryItemViewController: UIViewController {
+    public var queryItem: QueryItem?
+
+    public var delegate: QueryItemViewControllerDelegate?
+
+    public override func viewDidLoad() {
+        super.viewDidLoad()
+
+        switch queryItem?.query {
+        case .some(.booking(let bookingQuery)):
+            performSegue(withIdentifier: "bookingSegue", sender: bookingQuery)
+        case .some(.parking(let parkingQuery)):
+            performSegue(withIdentifier: "parkingSegue", sender: parkingQuery)
+        case .none:
+            Log("No QueryItem", data: nil, level: .error)
+            break
+        }
+    }
+
+    public override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        switch (segue.identifier, segue.destination, sender) {
+        case (_, let vc as BookingItemQueryViewController, let query as BookingItemQuery):
+            //vc.searchQuery = query
```

Comment:
should this be commented out?

Reply:
Yes, we don't have query items that execute Booking Queries that is there as a stub for when that happens.